lib/papertrail/cli.rb

```diff
@@ -95,6 +95,9 @@ def run
 
       @connection = Papertrail::Connection.new(options)
 
+      # Use HTTP Keep-Alive unless delay is too long
+      connection.start if options[:delay] < 10
```

Comment:
@jshawl no you are right I think: `<` would be a method on `nil` so this would blow up.

Reply:
It is initialized [here](https://github.com/papertrail/papertrail-cli/blob/d4ef7502e37fdf68a42f754c527343165b93475c/lib/papertrail/cli.rb#L19).